superflore/utils.py

```diff
@@ -202,7 +199,7 @@ def get_license(l):
     elif re.search(cc_re, l, f):
         return 'CC-BY-SA-3.0'
     elif re.search(boost_re, l, f):
-        return 'Boost-1.0'
```

Comment:
Please revert this change

Reply:
This still does not work with Gentoo, which gets very upset if this is not matched exactly.

Reply:
Done; bitbake is also happy about 'Boost-1.0'.